README.md

```diff
@@ -13,35 +13,118 @@ and [JaCoCo](https://github.com/jacoco/jacoco).
 
 ## Quickstart
 
-### Add repository
+### Add Maven Central repository
 
 For `settings.gradle`
 ```
 pluginManagement {
     repositories {
         // ... other dependencies
-        maven { url 'https://maven.pkg.jetbrains.space/public/p/jb-coverage/maven' }
+        mavenCentral()
     }
 }
 ```
 ### Apply plugin to project
-
+#### Applying plugins with the plugins DSL
 For `build.gradle.kts`
 ```
 plugins {
-     id("kotlinx-kover") version "0.3.0"
+     id("org.jetbrains.kotlinx.kover") version "0.3.0"
 }
 ```
 For `build.gradle`
 ```
 plugins {
-    id 'kotlinx-kover' version '0.3.0'
+    id 'org.jetbrains.kotlinx.kover' version '0.3.0'
+}
+```
+
+#### Legacy Plugin Application: applying plugins with the buildscript block
+For `build.gradle.kts`
+```
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath("org.jetbrains.kotlinx:kover:0.3.0")
+    }
+}
+
+apply(plugin = "org.jetbrains.kotlinx.kover")
+```
+
+For `build.gradle`
+```
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.jetbrains.kotlinx:kover:0.3.0'
+    }
 }
+  
+apply plugin: 'org.jetbrains.kotlinx.kover'    
```

Comment:
Can we still apply it with `kotlinx-cover`, as with `apply plugin: 'binary-compatibility-validator'` ?

Reply:
I can add an additional plugin id `kover` to the resources for legacy scripts